### DISCUSSION TITLE
error chowning path

### DISCUSSION THREAD

**Discussion by rashmimaharana** [drakkan/sftpgo#859] [Q&A] [answered]
Hi @drakkan and Team,

We have deployed sftpgo as container in GKE. Getting below error frequently but file transfer is success.

error chowning path transfer/sms_splits/incoming/upf/upf-ONLY-CNAONLY-84-20220530143931-5680eb85911a5da49d9147773e4c48d7.done: not supported

**Comment by drakkan** [comment, marked as answer]
Hi,

I guess you have set a UID/GID! = 0. In this case SFTPGo tries to change permissions for uploaded files. The GCS backend does not support changing file permissions

**Reply by rashmimaharana** [reply]
Thanks @drakkan . Yes we have set UID and GID a random value. Will revert that.
Thank you for your help.
